apps/web/src/views/Home/components/Banners/NemesisDownfallBanner.tsx

```diff
@@ -0,0 +1,106 @@
+import { useTranslation } from '@pancakeswap/localization'
+import { Box, Button, Flex, Image, Link, OpenNewIcon, Text, useMatchBreakpoints } from '@pancakeswap/uikit'
+import {
+  BackgroundGraphic,
+  BannerContainer,
+  BannerGraphics,
+  BannerMain,
+  BannerTitle,
+  FloatingGraphic,
+  GraphicDetail,
+  LinkExternalAction,
+  PancakeSwapBadge,
+} from '@pancakeswap/widgets-internal'
+import { ASSET_CDN } from 'config/constants/endpoints'
+import { styled } from 'styled-components'
+
+const floatingAsset = `${ASSET_CDN}/web/banners/nemesis-downfall/floating-item.png`
+const bgDesktop = `${ASSET_CDN}/web/banners/nemesis-downfall/bg-desktop.png`
+const bgMobile = `${ASSET_CDN}/web/banners/nemesis-downfall/bg-mobile.png`
+const logo = `${ASSET_CDN}/web/banners/nemesis-downfall/logo.png`
+const bgImage = `${ASSET_CDN}/web/banners/nemesis-downfall/background-image.png`
+
+const bgSmVariant: GraphicDetail = {
+  src: bgMobile,
+  width: 272,
+  height: 224,
+}
+
+const bgXsVariant: GraphicDetail = {
+  src: bgMobile,
+  width: 218,
+  height: 182,
+}
+
+const StyledButton = styled(Button)`
+  color: #ffffff;
+  background: #812b21;
+  box-shadow: inset 0px -2px 0px rgba(0, 0, 0, 0.2);
+
+  &:hover > a {
+    text-decoration: none;
+  }
+`
+
+export const NemesisDownfallBanner = () => {
+  const { t } = useTranslation()
+  const { isMobile, isXxl } = useMatchBreakpoints()
+
+  return (
+    <BannerContainer background={`url('${bgImage}')`}>
+      <BannerMain
+        badges={
+          <Flex>
+            <PancakeSwapBadge whiteText />
+            <Box style={{ alignSelf: 'center' }} ml="8px" width={isMobile ? 54 : 98} height={isMobile ? 11 : 20}>
+              <Image src={logo} alt="logo" width={isMobile ? 54 : 98} height={isMobile ? 11 : 20} />
+            </Box>
+          </Flex>
+        }
+        title={
+          <BannerTitle variant="orange">
+            {isMobile
+              ? t('Nemesis Downfall Now on PancakeSwap')
+              : t('Nemesis Downfall Now Live on PancakeSwap Gaming Marketplace')}
+          </BannerTitle>
+        }
+        actions={
+          isMobile ? (
+            <LinkExternalAction color="white" href="https://pancakeswap.games" style={{ alignItems: 'center' }}>
+              {t('Play Now')}
+            </LinkExternalAction>
+          ) : (
+            <>
+              <StyledButton scale={isXxl ? 'md' : 'sm'}>
+                <Link external href="https://pancakeswap.games">
+                  <Text
+                    bold
+                    mr="4px"
+                    color="white"
+                    fontSize={isMobile ? '12px' : '16px'}
+                    textTransform={isMobile ? 'uppercase' : 'capitalize'}
+                  >
+                    {t('Play Now')}
+                  </Text>
+                  <OpenNewIcon color="white" />
+                </Link>
+              </StyledButton>
+              <LinkExternalAction
+                ml="8px"
```

Comment:
It's better to have a `ActionContainer` component to handle the structure. `ActionContainer` can specify the flex gap so we don't need to worry about the style next time. Also the flex gap may be different on different devices
```jsx
import { ActionContainer } from 'widgets' 

<ActionContainer>
  <Button>Play now</Button>
  <LinkExternalAction>Learn more</LinkExternalAction>
</ActionContainer>
```